src/binomial.ts

```diff
@@ -120,12 +112,10 @@ function inversionCalc(u: number, n: number, p: number) {
     let f = Math.pow(q, n);
     let k = 0;
 
-    let fPrev = f;
```

Comment:
The linter flagged that apparently `fPrev` wasn't used. Is this expected?

Reply:
It's been a long time since I thought about this code, it's definitely a bit of a surprise. I don't see anything like this though in the equivalent C++ code

```
  while (u >= f) {
    u -= f;
    k++;
    f *= (g / k - r);
    if (k > max_k) {
      return -1;
    }
  }
```